Documentation/ContentObjects/Image/Index.rst

```diff
@@ -755,26 +786,25 @@ Responsive/adaptive rendering
    60.layoutKey = srcset
 
 
-This returns as an example all per default possible HTML-Output:
+This returns as an example all per default possible HTML output:
 
 .. code-block:: html
 
-   <img src="fileadmin/_processed_/imagefilenamename_595cc36c48.png"
-     width="600" height="423"    alt=""  border="0" />
-   <img src="fileadmin/_processed_/imagefilenamename_595cc36c48.png"
-     data-small="fileadmin/_processed_/imagefilenamename_595cc36c48.png"
-     data-smallRetina="fileadmin/_processed_/imagefilenamename_42fb68d642.png"
-     alt="" />
+   <img src="/fileadmin/_processed_/imagefilenamename_595cc36c48.png"
```

Comment:
When trying the code example above I only get this code, not the other ones below. Maybe I am doing something wrong. But this is not a stopper for this PR.